### PR TITLE
Add alfred-messages to Users section

### PR DESCRIPTION
**Pull request by briangonzalez**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -495,6 +495,7 @@ Non-synced local preferences are stored within `Alfred.alfredpreferences` under
 
 *Alfred workflows using Alfy*
 
+- [alfred-messages](https://github.com/briangonzalez/alfred-messages/issues) - Send messages to your contacts
```
Comment (SamVerschueren) [inline readme.md]: Add it at the bottom

Comment (briangonzalez) [inline readme.md]: done @SamVerschueren

Comment (SamVerschueren) [inline readme.md]: It points to your issue tracker.

**Comment by briangonzalez** [comment]
@SamVerschueren Any luck on this one sir?

**Comment by SamVerschueren** [comment]
Oh, I thought I already replied. You linked to the issue tracker, not the home page of the repo.

**Comment by briangonzalez** [comment]
Ok, try now. Sorry about that!

**Comment by briangonzalez** [comment]
@SamVerschueren This should be all set now. Thanks!

**Comment by SamVerschueren** [comment]
I noticed. Not entirely sure about the description, that's why I didn't merge yet. I didn't really knew what it did until I opened the github page.

**Comment by briangonzalez** [comment]
Would you like me to add a better description? Something like:

> Send messages (previously iMessages) to your contacts

**Comment by briangonzalez** [comment]
@SamVerschueren Let me know what I can do to move this one along. Thanks.

**Comment by SamVerschueren** [comment]
I'm awaiting the feedback from @sindresorhus.

**Comment by sindresorhus** [comment]
> previously iMessages

It's never been named `iMessages`. The service is called iMessage and the app is called Messages.

And the description is still not clear at all. I would use:

> Message your contacts through iMessage

**Comment by briangonzalez** [comment]
Fair enough. Thanks.

**Comment by briangonzalez** [comment]
@sindresorhus @SamVerschueren Here you go. Thanks.

**Pull request merged**